kinde_sdk/frontend/api/o_auth_api.py

```
@@ -0,0 +1,923 @@
+    Kinde Account API
+
+     Provides endpoints to operate on an authenticated user.  ## Intro  ## How to use  1. Get a user access token - this can be obtained when a user signs in via the methods you've setup in Kinde (e.g. Google, passwordless, etc).  2. Call one of the endpoints below using the user access token in the Authorization header as a Bearer token. Typically, you can use the `getToken` command in the relevant SDK. 
+
+    The version of the OpenAPI document: 1
+    Contact: support@kinde.com
+    Generated by OpenAPI Generator (https://openapi-generator.tech)
+
+    Do not edit the class manually.
+"""  # noqa: E501
+
+import warnings
```

Ruff (0.12.2)

`warnings` imported but unused

Remove unused import: `warnings`

(F401)